src/app/story/story.component.ts

```diff
@@ -79,10 +82,18 @@ export class StoryComponent implements OnInit, AfterViewInit, OnDestroy {
         const nodeDisplayNamePipe = new NodeDisplayNamePipe();
         const titleDisplay = nodeDisplayNamePipe.transform(this.selectedNode.display_name);
         this.titleService.setTitle(titleDisplay);
+        this.processingTimeFlag = story.meta.processing_time_flag;
+        this.getToast();
       }
     );
   }
 
+  getToast() {
+    if (this.processingTimeFlag) {
```

Comment:
Could you please make this check `=== true`? I don't like relying on truthy values in JavaScript since things like empty arrays will evaluate to true.